security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -108,6 +108,34 @@ Version `1.14.1` of the {cert-manager-operator} is based on the upstream cert-ma
 
 * link:https://access.redhat.com/security/cve/CVE-2024-33599[CVE-2024-33599]
 * link:https://access.redhat.com/security/cve/CVE-2024-2961[CVE-2024-2961]
+[id="cert-manager-operator-release-notes-1-14-2"]
```

Comment:
There needs to be a space between the ID and whatever comes before it, right now this heading is not showing up in the TOC or preview:
```suggestion

[id="cert-manager-operator-release-notes-1-14-2"]
```